google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ConnectionSettings.java

```diff
@@ -221,8 +221,7 @@ Builder withDefaultValues() {
      *
      * @param useReadAPI or {@code true} for none
      */
-    @Nullable
-    public abstract Builder setUseReadAPI(Boolean useReadAPI);
+    public abstract Builder setUseReadAPI(@Nullable Boolean useReadAPI);
```

Comment:
Done

Reply:
Sorry, I should have been more specific. What I meant was, we can leave getUseReadAPI() as:
  @Nullable
  public abstract Boolean getUseReadAPI();
  
  And update setUseReadAPI() to:
    public abstract Builder setUseReadAPI(Boolean useReadAPI);

Reply:
Thanks, I think I have done what you described, please take another look.

Can `UseReadAPI` ever by null? It's not obvious to me why it needs to be a `@Nullable Boolean` instead of a `boolean`.

Reply:
Thanks for the change.
`UseReadAPI` should normally not be null. However, `@Nullable Boolean` is preferred here as it is consistent with other values.